charts/drax/charts/golang-nkafka/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: golang-nkafka
 description: The Accelleran dRAX nats-kafka bridge
 type: application
-version: 2.0.0
-appVersion: 1.3.5
+version: 2.0.1
+appVersion: 1.3.7
```

Comment:
This isn't sufficient unfortunately. Due to the format of the version we'll need to inject it at `image.tag` (current value is `1.3.5_R4.3.8_leffe_CW3.0.1`.

One of the things we could also consider here is to just use proper semantic versions as I don't see the value of having different CU/CW versions included (given there will always only be 1 in the helm charts which is the only place where this is used?)

Reply:
To be discussed if needed: Once the telemetry collector is in place, this will become "legacy" and eventually, we will stop deploying it.